filebeat/input/filestream/copytruncate_prospector.go

```diff
@@ -230,7 +230,7 @@ func (p *copyTruncateFileProspector) Run(ctx input.Context, s loginp.StateMetada
 
 	errs := tg.Wait()
 	if len(errs) > 0 {
-		log.Error("%s", sderr.WrapAll(errs, "running prospector failed"))
+		log.Error("%s", fmt.Errorf("running prospector failed: %w", errors.Join(errs...)))
```

Comment:
This is redundant; the error is getting constructed and then immediately rendered into a string.
```suggestion
		log.Error("running prospector failed: %v", errors.Join(errs...))
```